subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -791,37 +670,33 @@ static bool pa_decode_base(struct bt_data *data, void *user_data)
 		}
 	}
 
+	base_size = bt_bap_base_get_size(base);
+
+	/* We provide the BASE without the service data UUID */
+	if (base_size < 0) {
+		LOG_DBG("BASE get size failed (%d)", base_size);
+
+		return false;
+	}
```

Comment:
Hmm, similar to `bt_bap_base_get_pres_delay`, this really can't (shouldn't) be able to fail if `bt_bap_base_get_base_from_ad` returns non-NULL. 

We can probably skip the verification part, or replace it with an assert.

Reply:
ok... so I am in favor of removing if it will never fail here (no need for unnecessary checks)